pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <artifactId>opensrp-server-web</artifactId>
     <packaging>war</packaging>
-    <version>2.5.13-SNAPSHOT</version>
+    <version>2.5.14-SNAPSHOT</version>
```

Comment:
This is a new backwards-compatible feature/endpoint. We can change to `2.6.0-SNAPSHOT`

Reply:
Updated.